pkgs/games/xonotic/default.nix

```diff
@@ -119,8 +119,9 @@ let
 
 in rec {
   xonotic-data = fetchzip {
-    name = "xonotic-data-${version}";
-    url = "https://dl.xonotic.org/${name}.zip";
+    name = "xonotic-data";
+    version = version;
```

Comment:
fetchzip doesn't take a version parameter (sorry for the 1-by-1 reviews, it's a big pr so I'm finding them mostly by building it)

Reply:
that is fine for me.